config/owasp/suppressions.xml

```diff
@@ -20,6 +20,16 @@
     <sha1>6936e700f0fb1b50bac0698ada4347a769d40199</sha1>
     <cpe>cpe:/a:apache:groovy</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[file name: tomcat-embed-core-9.0.17.jar - triggered by Gradle upgrade]]></notes>
```

Comment:
Can we not use the up to date imports?

Reply:
compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.19'